tempo_project/dataset/scripts/subsector.hs

```diff
@@ -0,0 +1,62 @@
+module Main where
+
+
+import Data.List (intercalate)
+
+
+main :: IO ()
+main = writeFile "bin.csv" . unlines . ("id,name" :) $ row <$> bins
+ 
+
+row :: (String, String, String, String, String) -> String
+row (c, i, u, z, t) = intercalate "+" [c, i, u, z, t] <> "," <> clean ("Composition = " <> c <> ", Income = " <> i <> ", Urbanity = " <> u <> ", Class = " <> z <> ", Technology = " <> t)
+
+clean :: String -> String
+clean = show . fmap (\x -> if x == '_' then ' ' else x)
+
+bins :: [(String, String, String, String, String)]
+bins = (,,,,) <$> composition <*> income <*> urbanity <*> clazz <*> tech
+
+composition :: [String]
+composition =
+  [
+    "No_Drivers"
+  , "Single_Driver"
+  , "Some_Drivers_Smaller"
+  , "Some_Drivers_Larger"
+  ]
+
+income :: [String]
+income =
+  [
+    "Low_Income"
+  , "Middle_Income"
+  , "High_Income"
+  ]
+
+urbanity :: [String]
+urbanity =
+  [
+    "Urban"
+  , "Suburban"
+  , "Second_City"
+  , "Small_Town"
+  , "Rural"
+  ]
+
+clazz :: [String]
+clazz =
+  [
+    "Compact"
+  , "Midsize"
+  , "SUV"
+  , "Pickup"
+  ]
+
+tech :: [String]
+tech =
+  [
+    "BEV_100"
+  , "BEV_300"
+  , "BEV_500"
```

Comment:
I believe this should be 
BEV_100
BEV_300
PHEV_25
PHEV_50
(see above tempo_project/dataset/dimensions/supplemental/tech.csv)

If this is fixed and re-run, it appears it will resolve the comment in tempo_project/dataset/scripts/make_dimension_mappings.py
# TODO: issue -- I can't find PHEV in the bin.csv ***************

Reply:
@ahcyip, is subsector.hs meaningful to you? Similarly, I don't know if anyone will be re-running make_dimension_mappings.py. Choices:

1. I make the changes you're suggesting in subsector.hs but don't test
2. You make the changes you're suggesting in subsector.hs and test the result
3. I delete subsector.hs and make_dimension_mappings.py

which do you prefer?

Reply:
I'm going to go with option 3, since these files are still in the original dsgrid_project folder and they seem irrelevant now.